pycbc/events/stat.py

```diff
@@ -539,14 +542,24 @@ def single(self, trigs):
     'newsnr_sgveto': NewSNRSGStatistic
 }
 
+sngl_statistic_dict = {
+    'newsnr': NewSNRStatistic,
+    'new_snr': NewSNRStatistic, # For backwards compatibility
+    'snr': NetworkSNRStatistic,
+    'newsnr_cut': NewSNRCutStatistic,
+    'exp_fit_csnr': ExpFitCombinedSNR,
+    'max_cont_trad_newsnr': MaxContTradNewSNRStatistic,
+    'newsnr_sgveto': NewSNRSGStatistic
+}
+
 def get_statistic(stat):
```

Comment:
might consider renaming this to get_coinc_statistic() .. that would clearly require a bunch of other changes though

Reply:
Yeah, I did consider that, but decided it wasn't worth the potential for breaking other code by renaming this. I can do this if you think its necessary though.

Reply:
Another day.